sdks/java/core/src/main/java/org/apache/beam/sdk/io/OffsetBasedSource.java

```diff
@@ -315,6 +330,30 @@ public Double getFractionConsumed() {
     }
 
     @Override
+    public long getParallelismConsumed() {
```

Comment:
Why not call this "getSplitsConsumed" or "getSplitPointsConsumed"? The Javadoc seemed to spend a lot of time explaining that "parallelism consumed" was related to how many splits had been consumed?